### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by MiaYWei**
1. Added confirmation when human player entered q or Q.
2. Removed the redundant deck display.
3. Added print info when computer dropped a skip card.
4. fixed test application compile issue.

### CODE REVIEW

**Review by aditibhardwaj98 — APPROVED** [review]

File: src/computer_player.c [hunk 1]
```diff
@@ -512,6 +513,7 @@ void process_playable_card(Card_t card)
     
     end_turn(COMPUTER);
     if (card.name == SKIP) {
+        print_info("COMPUTER discarded a Skip card, HUMAN will lose turn.\n");
```
Comment (krithigasr) [inline src/computer_player.c]: Can we change this to " Computer has dropped a skip card now, human player loses his turn" ?

File: src/human_player.c [hunk 2]
```diff
@@ -68,21 +68,15 @@ void invalid_card_warning(void)
...
         end_turn(HUMAN);
-        ret = RET_SUCCESS;
+        return RET_SUCCESS;
     }
-    else
-    {
-        print_warning("!!Warning!! Please draw a card before you can end your turn.\n");
-        display_player_deck(HUMAN);
+    else {
+        print_warning("!!Warning!! Please draw a card('n' or 'N'), then you can end your turn.\n");
```
Comment (krithigasr) [inline src/human_player.c:77]: Good addition here!

File: src/computer_player.c (hunk 1 above)
Comment (MiaYWei) [reply src/computer_player.c]: 1. COMPUTER and HUMAN are specified for the player "name", just want to keep it.  Here is hard coded, but in some other place we use "static const char* PLAYER_TYPE_STRING[] = { "HUMAN", "COMPUTER", "INVALID"};" If you want the change here, we all need to make the consistent change for the array. 
2. Avoid to use his, or we need to add her as well.

**Review by HongBo713 — APPROVED** [review]

**Review by krithigasr — APPROVED** [review]

File: src/computer_player.c (hunk 1 above)
Comment (MiaYWei) [reply src/computer_player.c]: Updated.

**Pull request merged**